mcmahon.py

```diff
@@ -347,28 +343,30 @@ def test_winner(self):
 class TournamentTestCase(unittest.TestCase):
 
     def setUp(self):
-        players = [Player('Ma Wang', 7, 12345, 6, 6, 1),
-                   Player('Will', 4, 1235, 6, 6, 1),
-                   Player('Steve', 4, 234, 6, 6, 1),
-                   Player('Mr. Cho', 3, 54, 6, 6, 1),
-                   Player('XiaoCheng', 3, 5723, 6, 6, 1),
-                   Player('Alex', 1, 632, 6, 6, 1),
-                   Player('Matt', 4, 5723, 6, 6, 1),
-                   Player('Ed', 2, 5723, 6, 6, 1),
-                   Player('Josh', 1, 5723, 6, 6, 1),
-                   Player('Kevin', 1, 5723, 6, 6, 1),
-                   Player('Gus', 1, 5723, 1, 1, 2),
-                   Player('Pete', -2, 5723, 1, 1, 2),
-                   Player('Dan', -1, 5723, 1, 1, 2),
-                   Player('David', -2, 5723, 1, 1, 2),
-                   Player('Alex', -3, 5723, 1, 1, 2),
-                   Player('Eric', -4, 5723, 1, 1, 2),
-                   Player('Makio', -4, 5723, 1, 1, 2),
-                   Player('David', -4, 5723, 1, 1, 2),
-                   Player('Eric', -4, 5723, 1, 1, 2),
-                   Player('Howie', -4, 5723, 1, 1, 2),
+        players = [Player('Ma Wang',    7, 12345, [6,0,0], 6, 1),
+                   Player('Will',       4, 1235,  [6,0,0], 6, 1),
+                   Player('Steve',      4, 234,   [6,0,0], 6, 1),
+                   Player('Mr. Cho',    3, 54,    [6,0,0], 6, 1),
+                   Player('XiaoCheng',  3, 5723,  [6,0,0], 6, 1),
+                   Player('Alex',       1, 632,   [6,0,0], 6, 1),
+                   Player('Matt',       4, 5723,  [6,0,0], 6, 1),
+                   Player('Ed',         2, 5723,  [6,0,0], 6, 1),
+                   Player('Josh',       1, 5723,  [6,0,0], 6, 1),
+                   Player('Kevin',      1, 5723,  [6,0,0], 6, 1),
+                   Player('Gus',        1, 5723,  [1,0,0], 1, 2),
+                   Player('Pete',      -2, 5723,  [1,0,0], 1, 2),
+                   Player('Dan',       -1, 5723,  [1,0,0], 1, 2),
+                   Player('David',     -2, 5723,  [1,0,0], 1, 2),
+                   Player('Alex',      -3, 5723,  [1,0,0], 1, 2),
+                   Player('Eric',      -4, 5723,  [1,0,0], 1, 2),
+                   Player('Makio',     -4, 5723,  [1,0,0], 1, 2),
+                   Player('David',     -4, 5723,  [1,0,0], 1, 2),
+                   Player('Eric',      -4, 5723,  [1,0,0], 1, 2),
+                   Player('Howie',     -4, 5723,  [1,0,0], 1, 2),
                    ]
         self.tournament = Tournament.new_tournament(players)
+        for player in self.tournament.players:
+            print(yaml.dump(self.tournament))
```

Comment:
Print statements like this shouldn't be committed as part of the unittest (even though they are super helpful while developing).